nima/webserver/webserver/src/main/java/io/helidon/nima/webserver/http/HttpRouting.java

```diff
@@ -319,22 +321,37 @@ public Builder any(String pattern, Handler handler) {
                                  .path(pattern)
                                  .handler(handler));
         }
+
+        /**
+         * Set max reroute count.
```

Comment:
The javadoc could be a bit more verbose.
Also do not use `Set` or `Get` verbs in javadoc.
Something like:
```
Maximal number of allowed re-routes within routing.

@see ServerResponse.reroute(String)
```

Reply:
done